ipt_connect/IPTdev/models.py

```diff
@@ -27,31 +27,21 @@ def mean(vec):
 		return 0
 
 def special_mean(vec):
-	if len(vec) in [5, 6]:
-		nreject = 1
-	elif len(vec) in [7, 8]:
-		nreject = 2
-	else:
-		nreject = round(len(vec) / 4.0)
+	nreject = round(len(vec) / 4.0)
 
-	if round(nreject / 2.0) == nreject / 2.0:
-		nlow = int(nreject / 2.0)
-		nhigh = int(nlow)
-	else:
+	if nreject % 2:
 		nlow = int(nreject / 2.0 + 0.5)
-		nhigh = int(nreject / 2.0 - 0.5)
+		nhigh = -int(nreject / 2.0 - 0.5) or None
```

Comment:
I don't really understand that part...  -int(nreject / 2.0 - 0.5) will always be a defined value, why the "or None" statement?

Reply:
Reverting the whole function to original version, so no need to think about this.